src/Sylius/Component/Shipping/Model/ShippingSubjectInterface.php

```diff
@@ -23,25 +23,25 @@
     /**
      * @return int
      */
-    public function getShippingWeight();
+    public function getShippingWeight(): int;
```

Comment:
I guess we have a mismatch in storing weight / depth etc. Once at a time it's a float, another time it's an integer.